.github/workflows/windows.yml

```diff
@@ -66,16 +66,16 @@ jobs:
         run: |
           New-Item build -ItemType Directory
           Set-Location build
-          qmake -config ${{ matrix.build-type }} ..\src\Notes.pro
-          nmake
+          cmake .. -G Ninja -DCMAKE_BUILD_TYPE=${{ matrix.build-type }}
```

Comment:
Does the `-B` switch not work on Windows? 🤔 

Just so we can get rid of the `New-item` / `Set-Location` lines above this one.

Reply:
I believe I tried this and it stopped working once we switched the minimum cmake version to 3.10 in `CMakeLists.txt`. 

I'll test this again to be sure.

Reply:
Looks like it does work :)